generator/generator.ml

```diff
@@ -683,7 +696,7 @@ let check_functions () =
   (* Check short descriptions. *)
   List.iter (
     fun (name, _, shortdesc, _) ->
-      if shortdesc.[0] <> Char.lowercase shortdesc.[0] then
+      if shortdesc.[0] <> Char.lowercase_ascii shortdesc.[0] then
```

Comment:
?

Reply:
lowercase is no longer compatible with latest ocaml version, its now lowercase_ascii